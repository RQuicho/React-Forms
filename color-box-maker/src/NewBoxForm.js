import {React, useState} from 'react';

// const NewBoxForm = () => {
//   return (

//   )
// }

export default NewBoxForm;