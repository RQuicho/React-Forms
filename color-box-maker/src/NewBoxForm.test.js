import React from 'react';
import {render, fireEvent} from '@testing-library/react';
import NewBoxForm from './NewBoxForm';

// smoke test
it('renders without crashing', () => {
  render (<NewBoxForm />);
})

// snapshot test
it('matches snapshot', () => {
  const {asFragment} = render(<NewBoxForm />);
  expect (asFragment()).toMatchSnapshot();
})