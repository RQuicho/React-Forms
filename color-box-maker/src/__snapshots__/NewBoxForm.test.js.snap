// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <form>
    <label
      for="color"
    >
      Color
    </label>
    <input
      id="color"
      name="color"
      placeholder="color"
      type="text"
      value=""
    />
    <label
      for="width"
    >
      Width
    </label>
    <input
      id="width"
      name="width"
      placeholder="0"
      type="number"
      value=""
    />
    <label
      for="height"
    >
      Height
    </label>
    <input
      id="height"
      name="height"
      placeholder="0"
      type="number"
      value=""
    />
    <button>
      Add Box
    </button>
  </form>
</DocumentFragment>
`;
